control-plane/csi-driver/src/bin/node/error.rs

```diff
@@ -145,3 +148,27 @@ impl From<DeviceError> for tonic::Status {
         }
     }
 }
+
+/// Error for the fsfreeze operations.
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(crate)), context(suffix(false)))]
+pub(crate) enum FsfreezeError {
+    #[snafu(display("Cannot find volume: volume ID: {}", volume_id))]
+    VolumeNotFound { volume_id: String },
+    #[snafu(display("Volume is not mounted: volume ID: {}", volume_id))]
+    VolumeNotMounted { volume_id: String },
+    #[snafu(display("Invalid volume ID: {}, {}", volume_id, source))]
+    InvalidVolumeId {
+        source: uuid::Error,
```

Comment:
Depending on how errors are returned up, ensure source error doesn't appear twice in the error message.

Reply:
I don't think they are, this is being returned at one place only.